site/package.json

```diff
@@ -33,7 +33,7 @@
     "@material-ui/lab": "4.0.0-alpha.42",
     "@testing-library/react-hooks": "8.0.1",
     "@xstate/inspect": "0.6.5",
-    "@xstate/react": "3.0.0",
+    "@xstate/react": "3.0.1",
```

Comment:
What was the testing issue this fixed?

Reply:
The `getSnapshot` warning/crash and the `shallowEqual` util for avoiding it. https://newreleases.io/project/github/statelyai/xstate/release/@xstate%2Freact@3.0.1